.github/workflows/build.yml

```diff
@@ -5,8 +5,14 @@ name: Build APK
 # Controls when the action will run. Triggers the workflow on push or pull request events
 on:
   push:
+    paths-ignore:
+      - '*.md'
+      - '*.txt'
```

Comment:
As suggested by @SuperSandro2000 I would exclude .txt as this could create side-effect
```suggestion
```